.github/actions/maven-build-and-tag/action.yml

```diff
@@ -258,7 +258,7 @@ runs:
         echo "TESTCONTAINERS_RYUK_DISABLED=true" >> $GITHUB_ENV
 
     - name: Set SONAR_SCANNER_OPTS
-      if: inputs.sonar-token != '' && env.SKIP_TESTS != 'true'
+      if: inputs.sonar-token != '' && env.SKIP_TESTS != 'true' && github.event.pull_request.draft == false
```

Comment:
Why isn't checking SKIP_TESTS sufficient? Aren't our PRs opened with the skip-tests label by default?